consensus/module.go

```diff
@@ -260,14 +266,15 @@ func (m *consensusModule) GetModuleName() string {
 }
 
 func (m *consensusModule) GetBus() modules.Bus {
-	if m.bus == nil {
+	bus := m.IntegratableModule.GetBus()
```

Comment:
In here the `GetBus` can be removed. The `SetBus` can't and we have to call `IntegratableModule` because it's like `super()` in OOP derived classes.